composer.json

```diff
@@ -9,7 +9,7 @@
         "cakephp/cakephp": "3.4.*",
         "mobiledetect/mobiledetectlib": "2.*",
         "cakephp/migrations": "~1.0",
-        "cakephp/plugin-installer": "*"
+        "cakephp/plugin-installer": "~0.0.15"
```

Comment:
@liviascapin can you change it to ``~1.0``, please? As @markstory [mentioned](https://github.com/cakephp/app/pull/502#issuecomment-287834561).